### PR TITLE
Add filter to the users dropdown and list of selected members

### LINKED ISSUE #12438: Add filter ability to the add member overlay

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #12438 

_Briefly describe your proposed changes:_
Create filter on the add member overlay and add member list to display selected members before saving changes.

  - [x] Rebased/mergeable
  - [x] Tests pass

Kapture 2019-03-11 at 12 24 37

### CODE REVIEW

File: ui/src/organizations/components/AddMembersForm.tsx [hunk 1]
```diff
@@ -1,31 +1,55 @@
...
-import {Button, ButtonType} from '@influxdata/clockface'
-import {Form, Grid} from 'src/clockface'
+import {Button, ButtonType, IconFont} from '@influxdata/clockface'
+import {Form, Grid, Input, Columns} from 'src/clockface'
 import SelectUsers from 'src/organizations/components/SelectUsers'
 
 // Types
-import {UsersMap} from 'src/organizations/components/Members'
+import {User} from '@influxdata/influx'
+import {UsersMap} from './Members'
```
Comment (ebb-tide) [inline ui/src/organizations/components/AddMembersForm.tsx]: I think the abs. path is better in this case? 'src/organizations/components/Members'

File: ui/src/organizations/components/AddMembersOverlay.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 // Components
 import {OverlayBody, OverlayHeading, OverlayContainer} from 'src/clockface'
 import AddMembersForm from './AddMembersForm'
-import {AddResourceMemberRequestBody} from '@influxdata/influx'
+import {AddResourceMemberRequestBody, User} from '@influxdata/influx'
 
 // Types
 import {UsersMap} from 'src/organizations/components/Members'
+import FilterList from 'src/shared/components/Filter'
```
Comment (ebb-tide) [inline ui/src/organizations/components/AddMembersOverlay.tsx]: filter list can go under components

File: ui/src/organizations/components/AddMembersOverlay.tsx [hunk 3]
```diff
@@ -1,13 +1,14 @@
 // Libraries
-import React, {PureComponent} from 'react'
+import React, {PureComponent, ChangeEvent} from 'react'
 
 // Components
 import {OverlayBody, OverlayHeading, OverlayContainer} from 'src/clockface'
 import AddMembersForm from './AddMembersForm'
-import {AddResourceMemberRequestBody} from '@influxdata/influx'
+import {AddResourceMemberRequestBody, User} from '@influxdata/influx'
```
Comment (ebb-tide) [inline ui/src/organizations/components/AddMembersOverlay.tsx]: these can go under types

File: ui/src/organizations/components/AddMembersForm.tsx (hunk 1 above)
Comment (Palakp41) [reply ui/src/organizations/components/AddMembersForm.tsx]: I agree!

File: ui/src/organizations/components/AddMembersOverlay.tsx (hunk 3 above)
Comment (Palakp41) [reply ui/src/organizations/components/AddMembersOverlay.tsx]: True :)

File: ui/src/organizations/components/AddMembersOverlay.tsx (hunk 2 above)
Comment (Palakp41) [reply ui/src/organizations/components/AddMembersOverlay.tsx]: Yup :)

**Pull request merged**